mix/qml.qrc

```diff
@@ -10,6 +10,28 @@
         <file>qml/ProjectList.qml</file>
         <file>qml/StateDialog.qml</file>
         <file>qml/StateList.qml</file>
+        <file>qml/js/main.js</file>
```

Comment:
I suggest using consistent style for file names. We already use PascalCase for cpp/h, qml and js files. alllowercase for images is ok I think, although looks ugly